### PR TITLE
website: Update input group documentation & demos

### PR DESCRIPTION
**Pull request by idakukimiya**
- Update documentation, include adding hyperlinks to mentioned components.
- Remove placeholder

### CODE REVIEW

File: apps/website/src/pages/docs/inputgroup.mdx [hunk 1]
```diff
@@ -9,17 +9,18 @@ group: inputs
...
 
-<Placeholder componentPageName='input-inputgroup' />
-
 <LiveExample src='InputGroup.main.tsx'>
   <AllExamples.InputGroupMainExample client:load />
 </LiveExample>
 
+Input group class is a container to enhance an input by adding an icon, text, or a button in front or behind the input field as a "help text".
+
+Multiple add-ons are supported and can be mixed with [checkbox](checkbox), [radio](radio), [toggle switch](toggleswitch) input version. To learn more about when to use a toggle switch or a checkbox, please read [Checkbox vs Toggle Switch by Saadia Minhas](https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8).
```
Comment (gretanausedaite) [inline apps/website/src/pages/docs/inputgroup.mdx:22]: Remove link to "Checkbox vs Toggle Switch by Saadia Minhas" article. It's already referenced in both, checkbox and toggle switch.

Comment (elephantcatdog) [reply apps/website/src/pages/docs/inputgroup.mdx]: Hi, Ida, you should let the person who made the review resolve the issue. In this case, while you removed the text, you didn't remove the actual link. You'll need to remove the entire sentence "To learn more about when..."
image

**Comment by elephantcatdog** [comment]
Can you please add the examples "Checkbox group" and "Toggle group" from storybook?
image
image

File: apps/website/src/pages/docs/inputgroup.mdx (hunk 1 above)
Comment (idakukimiya) [reply apps/website/src/pages/docs/inputgroup.mdx]: fix

File: apps/website/src/pages/docs/inputgroup.mdx [hunk 2]
```diff
@@ -9,15 +9,22 @@ group: inputs
...
 
-<LiveExample src='InputGroup.main.tsx'>
-  <AllExamples.InputGroupMainExample client:load />
+<LiveExample src='InputGroup.toggleSwitch.tsx'>
+  <AllExamples.InputGroupToggleSwitchExample client:load />
+</LiveExample>
+
+## Usage
+
+Multiple add-ons are supported and can be mixed with [checkbox](checkbox), [radio](radio), [toggle switch](toggleswitch) input version.
```
Comment (gretanausedaite) [inline apps/website/src/pages/docs/inputgroup.mdx:24]: Add `RelatedComponents` links to checkbox, radio and toggle switch

File: apps/website/src/pages/docs/inputgroup.mdx [hunk 3]
```diff
@@ -9,17 +9,24 @@ group: inputs
...
 
-<LiveExample src='InputGroup.main.tsx'>
-  <AllExamples.InputGroupMainExample client:load />
+## Usage
+
+<LiveExample src='InputGroup.checkboxGroup.tsx'>
+  <AllExamples.InputGroupCheckboxGroupExample client:load />
 </LiveExample>
 
+Multiple add-ons are supported and can be mixed with [checkbox](https://itwinui.bentley.com/docs/checkbox#with-input-group), [radio](https://itwinui.bentley.com/docs/radio#statuses), [toggle switch](https://itwinui.bentley.com/docs/toggleswitch#with-input-group) input version.
```
Comment (gretanausedaite) [inline apps/website/src/pages/docs/inputgroup.mdx:28]:
```suggestion
Multiple add-ons are supported and can be mixed with [checkbox](checkbox#with-input-group), [radio](radio#statuses), [toggle switch](toggleswitch#with-input-group) input version.
```

**Comment by elephantcatdog** [comment]
The checkbox example looks nice! I'm not sure what happened with my last comment; it looks like there was already the toggle example. I meant to ask you to include the radio group example, like you've mentioned in the text.
image

File: apps/website/src/examples/InputGroup.radioGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+          value='option2'
+          onChange={action('Clicked option 2!')}
+          label={option2Label}
+          disabled={args.disabled}
+          required={args.required}
+        />
+      </InputGroup>
+    </>
+  );
+};
```
Comment (elephantcatdog) [inline apps/website/src/examples/InputGroup.radioGroup.tsx:39]: So the website is broken when I try to view this. The issue is with the import for `SvgSmileyHappy`. It also needs to be imported the same way that the sad face is. There's also some extra things from storybook that aren't needed in this example.

I think you can just change this example to this:
```suggestion
/*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
import { InputGroup, Radio } from '@itwin/itwinui-react';
import { SvgSmileyHappy, SvgSmileySad } from '@itwin/itwinui-icons-react';

export default () => {
  return (
    <>
      <InputGroup label='Radio group' message='Tell me how happy you are'>
        <Radio name='choice' value='option1' label={<SvgSmileyHappy />} />
        <Radio name='choice' value='option2' label={<SvgSmileySad />} />
      </InputGroup>
    </>
  );
};
```

**Review by gretanausedaite — APPROVED** [review]

**Pull request merged**